rootfs/api/tests/test_pods.py

```diff
@@ -415,6 +415,8 @@ def test_scale_errors(self, mock_requests):
         body = {'web': -1}
         response = self.client.post(url, body)
         self.assertEqual(response.status_code, 400, response.data)
+        self.assertEqual(response.data, {'detail': "Invalid scaling format: "
+                                         "['Must be greater than or equal to zero']"})
```

Comment:
Yes, it just looks confusing. the key is `detail`, the value is a string: `Invalid scaling format: ['Must be greater than or equal to zero']`. It just looks off here,